css/main.css

```diff
@@ -32,6 +32,9 @@ a         { color: #00a; }
 a:hover   { color: #000; }
 a:visited { color: #a0a; }
 
+img {
+  max-width: 100%;	
```

Comment:
I think in this case we also want a "height: auto;" to enable proportional scaling. What do you think?